### ISSUE TITLE
Fatal Exception: java.lang.ArrayIndexOutOfBoundsException: length=0; index=3

### ISSUE THREAD

**Issue by GianfrancoMS** [signalfx/splunk-otel-android#359]
We're using version 0.16.0 and we're getting some crashes at SlowRenderingDetectorImpl:

```
Fatal Exception: java.lang.ArrayIndexOutOfBoundsException: length=0; index=3
       at androidx.core.app.FrameMetricsAggregator$FrameMetricsApi24Impl$1.onFrameMetricsAvailable(FrameMetricsAggregator.java:368)
       at android.view.FrameMetricsObserver.onFrameMetricsAvailable(FrameMetricsObserver.java:59)
       at android.graphics.HardwareRendererObserver.lambda$notifyDataAvailable$0$HardwareRendererObserver(HardwareRendererObserver.java:93)
       at android.graphics.HardwareRendererObserver$$ExternalSyntheticLambda0.run(HardwareRendererObserver.java:2)
       at android.os.Handler.handleCallback(Handler.java:978)
       at android.os.Handler.dispatchMessage(Handler.java:104)
       at android.os.Looper.loopOnce(Looper.java:238)
       at android.os.Looper.loop(Looper.java:357)
       at android.os.HandlerThread.run(HandlerThread.java:67)
```

**Comment by GianfrancoMS** [comment]
It looks very similar to an issue I reported sometime ago https://github.com/signalfx/splunk-otel-android/issues/302

**Comment by mateuszrzeszutek** [comment]
Hey @GianfrancoMS ,
Thanks for submitting the issue -- I think we'll get it right this time with #361.
We'll include this fix in the 0.18 release.
